packages/devtools_app/lib/src/standalone_ui/vs_code/debug_sessions.dart

```diff
@@ -191,4 +182,16 @@ class _DevToolsMenu extends StatelessWidget {
       ),
     );
   }
+
+  bool _shouldIncludeScreen(ScreenMetaData screen) {
+    // Don't show home screen or debugger in the menu.
+    return screen != ScreenMetaData.home &&
+        screen != ScreenMetaData.debugger &&
+        // Or the generic "simple" screen.
```

Comment:
do we also need to hide the provider screen here? We might also want to hide `vmTools` and `deepLinks`.

`vmTools` should be gated behind a DevTools setting `preferences.vmDeveloperModeEnabled`. Since we have the server available we could check for that setting. Or we can just hide for now since that screen is intended for VM authors.

`deepLinks` is behind an experiment flag, so we should check that `FeatureFlags.deepLinkValidation` is true before including it.

Reply:
> do we also need to hide the provider screen here?

It's hidden by the `requiresLibrary` check further down. We could add it explicitly, but right now we don't support any screens that have library requirements (and I expect we don't bother, and instead will support them via extensions later).

I've hidden `vmTools` for now (to avoid making async calls out to check this), and also added a check for the feature flag for deep links (I meant to ask about this - I noticed it was showing a TODO when I clicked it!).